lib/config.dart

```diff
@@ -24,15 +25,28 @@ class Config {
       _log.v("Creating Config instance");
       final injector = serviceInjector ?? ServiceInjector();
       final breezLib = injector.breezLib;
-      String configString = await rootBundle.loadString('conf/breez.conf');
-      ini.Config breezConfig = ini.Config.fromString(configString);
-      final defaultConf = await breezLib.defaultConfig(sdk.EnvironmentType.Production);
+      final breezConfig = await _getBundledConfig();
+      final defaultConf = await _getDefaultConf(breezLib);
       final sdkConfig = await getSDKConfig(injector, defaultConf, breezConfig);
       _instance = Config._(sdkConfig);
```

Comment:
What do you think about adding another field `defaultConfig` to the Config class which contains the merge of the bundle with the default sdk?
This way we will not need to load and parse it again when calling `getMempoolSpaceDefaultUrl`